tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+
+		it('should return a promise resolving to a terms of service object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'get').returns(Promise.resolve(response));
+			return termsOfService.get(TERMS_OF_SERVICE_ID)
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+
+	describe('getAll()', function() {
+		it('should make GET request to get terms of service info when called', function() {
+			var response = {
+				total_count: 2,
+				entries: [{
+					type: 'terms_of_service',
+					id: '1234',
+					status: 'enabled',
+					enterprise: {},
+					tos_type: 'external',
+					text: 'TEST',
+					created_at: '2017-10-21T20:09:59-07:00',
+					modified_at: '2017-10-21T20:09:59-07:00'
+				}]
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('get').withArgs('/terms_of_services')
+				.returns(Promise.resolve(response));
+			termsOfService.getAll();
+		});
+
+		it('should call callback with the terms of service info when a 200 response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'get').yieldsAsync(null, response);
+			termsOfService.getAll(null, function(err, termsOfServiceObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to a terms of service object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'get').returns(Promise.resolve(response));
+			return termsOfService.getAll()
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+	describe('create()', function() {
+		var expectedTermsOfServiceParam = { body: { status: 'enabled', text: 'Test Text', tos_type: 'managed'}};
```

Comment:
It looks like you're just using this in the first test, so you can pull it into that `it` block